### PR TITLE
[Pay in Person] Change "Pay in Person" error text and illustration

### LINKED ISSUE #10361: Change "Pay in Person" error text and illustration
The current wording on the 'Pay in Person' error screen may lead to confusion. It suggests that activating the Pay in Person feature is required to collect IPP for orders created in-app, which is not the case. We need to revise both the text and the illustration to clarify that enabling Pay in Person is an optional choice for our customers.

More context: p1700477749579599-slack-C025A8VV728

### PR DESCRIPTION
**Pull request by AnirudhBhat**
Closes: #10361 

### Description

This PR changes the illustration and the error hint to better convey that Pay in Person is not mandatory to collect IPP for orders created in-app.

Design: w3JBReW1UJesCf02sj43kR-fi-1445%3A15221

### Testing instructions

1. Disable COD option either in the web admin or from the payments hub screen
2. Navigate to the payments hub screen
3. Ensure you see a message at the bottom which indicates that the onboarding is not complete
4. Click on it
5. Ensure you see "Pay in Person" error page
6. Ensure that the illustration and the text are as per design.
7. Ensure the error screen looks fine in dark mode

## Screenshots

**Before Changes** | **After Changes**
:-----------------:|:-----------------:
Before | After

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1663,12 +1663,15 @@
 
     <string name="card_reader_onboarding_learn_more">&lt;a href=\'\'&gt;Learn more&lt;/a&gt; about accepting payments with your mobile device and ordering card readers</string>
     <string name="card_reader_onboarding_contact_support">Need some help? &lt;a href=\'\'&gt;Contact support&lt;/a&gt;</string>
+    <string name="card_reader_onboarding_contact_us">Need some help? &lt;a href=\'\'&gt;Contact us&lt;/a&gt;</string>
 
     <string name="card_reader_onboarding_generic_error_header">Unable to verify In-Person Payments for this store</string>
     <string name="card_reader_onboarding_generic_error_hint">We\'re sorry, we were unable to verify In-Person Payments for this store</string>
 
     <string name="card_reader_onboarding_cash_on_delivery_disabled_error_header">Add Pay in Person to your checkout</string>
```
Comment (AnirudhBhat) [inline WooCommerce/src/main/res/values/strings.xml]: ❓ Should we remove the old string resource in all languages or just the english one is enough?

Comment (kidinov) [reply WooCommerce/src/main/res/values/strings.xml]: Tried to recall, but no luck. Probably worth asking the platform team

**Review by kidinov — COMMENTED** [review]
Code wise all looks good

* The icon we used before had light/dark versions. The new one is the only one. It looks ok, but differently compared to the other's error screens
* Now we have more content on this screen, but still keep top margin and on the small screens it doesn't fit
* In the bottom, there is a weird padding, so again, part of the screen is no in use


https://github.com/woocommerce/woocommerce-android/assets/4923871/56ff9167-51ea-41eb-91de-170636daab50

**Comment by AnirudhBhat** [comment]
Thanks for the review @kidinov
> Now we have more content on this screen, but still keep top margin and on the small screens it doesn't fit

I've removed the top margin here: https://github.com/woocommerce/woocommerce-android/pull/10362/commits/46f7c4f7972e88fde87255a1ca687c5a954cfabf

> The icon we used before had light/dark versions. The new one is the only one. It looks ok, but differently compared to the others errors

Agree, I've asked this to Joe in Slack and he said he will provide one tomorrow - p1702351384728189/1700477749.579599-slack-C025A8VV728

> In the bottom there is a weird padding, so again, part of the screen is no in use

This is because of the `verticalPadding` inside `ScrollView` in `fragment_card_reader_onboarding.xml`. That's probably added for ensuring that the content within the ScrollView does not touch the edges of the screen

File: WooCommerce/src/main/res/values/strings.xml (hunk 1 above)
Comment (AnirudhBhat) [reply WooCommerce/src/main/res/values/strings.xml]: Looks like we can modify the existing string and the changes gets picked up automatically - p1507221278000008-slack-C02QANACA

I have removed the new string resources and replaced existing string resources with the new text: https://github.com/woocommerce/woocommerce-android/pull/10362/commits/0e103f9bc92cc9a7c7313898b39d516c4e956e6f

**Comment by kidinov** [comment]
@AnirudhBhat 

> This is because of the verticalPadding inside ScrollView in fragment_card_reader_onboarding.xml. That's probably added for ensuring that the content within the ScrollView does not touch the edges of the screen

Oh yeah, I remember hitting this already some time ago. To fix that, we need to adjust all the included screens, which probably not worth it

**Review by kidinov — APPROVED** [review]
LGTM!

**Pull request merged**